cmudb/tscout/clang_parser.py

```diff
@@ -177,6 +189,11 @@ def __init__(self):
             for node_name in self._bases
         }
 
+        self.enum_map: Mapping[str, List[Tuple]] = {
```

Comment:
AFAIK that tuple is a str and an int?

```suggestion
        self.enum_map: Mapping[str, List[Tuple[str, int]]] = {
```